lmcache/integration/vllm/vllm_adapter.py

```diff
@@ -643,9 +674,11 @@ def lmcache_retrieve_kv(
             lmc_num_computed_tokens_list.append(lmc_num_computed_tokens)
 
             # No cache found, move on
-            if lmc_num_computed_tokens == 0:
+            if lmc_num_computed_tokens == 0 or seq_hidden_states is None:
```

Comment:
For the `non-MLA` case, maybe it did not store the `seq_hidden_states` at all, so it should be None as expected?